.gitignore

```diff
@@ -5,4 +5,16 @@
 /go
 ebpf-profiler
 ci-kernels
-/target
+# Ignore target directory
+target/*
+# But not these specific paths
+!target/x86_64-unknown-linux-musl/
+target/x86_64-unknown-linux-musl/*
+!target/x86_64-unknown-linux-musl/release/
+target/x86_64-unknown-linux-musl/release/*
+!target/x86_64-unknown-linux-musl/release/libsymblib_capi.a
+!target/aarch64-unknown-linux-musl/
+target/aarch64-unknown-linux-musl/*
+!target/aarch64-unknown-linux-musl/release/
+target/aarch64-unknown-linux-musl/release/*
+!target/aarch64-unknown-linux-musl/release/libsymblib_capi.a
```

Comment:
Is this required (maybe i don't understand the exact purpose here)? Once you `git add -f` the `*.a` files, `git status` they are excluded from `target/` (yes, `/target` should indeed be `target/`.

Reply:
Right, these shouldn't be needed. Our current `.gitignore` works.

Reply:
I didn't want to use `-f` when adding the binary blobs to git.
But as this seems to be ok, I have dropped the intermediate paths with https://github.com/open-telemetry/opentelemetry-ebpf-profiler/pull/415/commits/5705944447ad679c7563d768e58477b32b57da39